example/signature/app.go

```diff
@@ -157,6 +157,7 @@ func main() {
 	if err != nil {
 		log.Fatalf("Error initializing signature auth: %v", err)
 	}
+	config.ServiceAuthRequests = signatureAuth
 
```

Comment:
As discussed previously we would like to avoid this type of circular dependency. This should be addressed in #38